src/client/app/component/BusinessCard/index.js

```diff
@@ -15,7 +15,8 @@ class BusinessCard extends React.Component {
   render() {
     let data = this.props.itemInfo;
     let score = Math.floor(data.rating);
-    let style = {'background-image': 'url(' + data.imageUrl + ')'};
+    let imagesUrl = data.imageUrl || data.images[0];
```

Comment:
This is only because we still have some data with the previous structure, right? If so then we can say that those data are bad and we can just remove them.